.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -23,9 +23,11 @@
 <!-- Link any related issues, pull-requests, or commit hashes that are relevant to this pull request. If you are opening a PR without a corresponding issue please consider creating one first, at https://github.com/k3s-io/k3s/issues . A functional example will greatly help QA with verifying/reproducing a bug or testing new features. -->
 
 #### User-Facing Change ####
+<!-- If there are User-Facing changes, follow the instructions below, if not, leave the block empty. -->
 <!--
-Does this PR introduce a user-facing change? If no, just write "NONE" in the release-note block below.
-If the PR requires additional action from users switching to the new release, include the string "action required".
+    Each change should be in a new line;
+    The first character of each line must be a letter;
+    If the PR requires additional action from users switching to the new release, include the string "action required";
```

Comment:
Can we keep the note about `NONE`, assuming that still prevents the PR from showing up in the release notes?
```suggestion
    Each change note should be on a line by itself.
    Each change note should be a complete sentence, starting with a capitalized word and ending with punctuation.
    If the change requires additional action from users switching to the new release, include the string "action required" in the note.
    If the change has no user impact or should otherwise be hidden from the release notes, use the bare word NONE as the note.
```

Reply:
Most PRs don't adhere to this, and some write it outside the block or a different text.  The `gen_release_notes` command will ignore the block if it's empty, so I think it makes sense to remove it, here are some examples:
#7873, #7872, #7858, #7839, #7834, #7833, #7827, #7805, #7777, #7776, #7259, #7153, #6998

Reply:
I think it's easier for all involved to have a literal nothing than a word to indicate nothing. Less complicated tooling then.

Reply:
I believe we need some way to exclude things from the release notes. I don't have strong feelings on whether that is an opt-out, or opt-in, but the special handling for NONE was documented in the PR template and was intended as a way to prevent the release notes from being polluted with meaningless changes, such as pure CI/dev  changes that do not have any impact on end users.